docs/sql/system.txt

```diff
@@ -688,6 +688,65 @@ _node
 |                      | table.                           |             |
 +----------------------+----------------------------------+-------------+
 
+recovery
+--------
+
+Recovery happens whenever an index shard is moved to a different node or loaded
```

Comment:
wording: index -> table

maybe something like: Recovery is the process of ... e.g. during node startup, ...